fvtest/porttest/omrslTest.cpp

```diff
@@ -270,3 +280,39 @@ TEST(PortSlTest, sl_AixDLLWrongPlatform)
 	reportTestExit(OMRPORTLIB, testName);
 }
 #endif /* defined(AIXPPC) */
+
+#if defined(J9ZOS39064)
+/**
+ * Basic test: load the port Library 31-bit test dll via CEL4RO31 interface
+ */
+TEST(PortSlTest, sl_testOpen31bitDLLviaCEL4RO31)
+{
+	OMRPORT_ACCESS_FROM_OMRPORT(portTestEnv->getPortLibrary());
+	const char *testName = "omrsl_testOpen31bitDLLviaCEL4RO31";
+	uintptr_t dllHandle;
+	uintptr_t funcHandle;
+	uintptr_t rc = 0;
+	char sharedLibName[] = "sltestlib31";
+	char functionName[] = "sl_testOpen31bitDLLviaCEL4RO31_function";
+	const char *functionArgs = "()V";
+
+	reportTestEntry(OMRPORTLIB, testName);
+
+	if ((NULL != CEL4RO31_FNPTR) && (NULL != CELQGIPB_FNPTR)) {
+		/* Only attempt the test if the underlying LE support is available. */
+		rc = omrsl_open_shared_library(sharedLibName, &dllHandle, OMRPORT_SLOPEN_DECORATE | OMRPORT_SLOPEN_ATTEMPT_31BIT_OPEN);
+		if (0 != rc) {
+			outputErrorMessage(PORTTEST_ERROR_ARGS, "Unable to open %s, \n", sharedLibName, omrerror_last_error_message());
+			goto exit;
+		}
+		rc = omrsl_lookup_name(dllHandle, functionName, &funcHandle, functionArgs);
+		if (0 != rc) {
+			outputErrorMessage(PORTTEST_ERROR_ARGS, "Unable to lookup function %s from library %s, \n", functionName, sharedLibName, omrerror_last_error_message());
```

Comment:
ref: https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#function-parameter-list-does-not-fit-on-one-line
```suggestion
			outputErrorMessage(
				PORTTEST_ERROR_ARGS, "Unable to lookup function %s from library %s, \n",
				functionName, sharedLibName, omrerror_last_error_message());
```